UnleashClient/deprecation_warnings.py

```diff
@@ -10,7 +10,7 @@ def strategy_v2xx_deprecation_check(strategies: list) -> None:
     for strategy in strategies:
         try:
             # Check if the __call__() method is overwritten (should only be true for custom strategies in v1.x or v2.x.
-            if strategy.__call__ != Strategy.__call__:
+            if strategy.__call__ != Strategy.__call__:  # type:ignore
```

Comment:
Is this linter stuff as well?

Reply:
This is the type checker, it's gotten a little confused and can't correctly infer the type of `Strategy`. I think this is okay